product/roundhouse.tasks/Roundhouse.cs

```diff
@@ -30,7 +41,7 @@ public sealed class Roundhouse : ITask, ConfigurationPropertyHolder
         bool ITask.Execute()
         {
             run_the_task();
-            return true;
+            return !this.loggingHelper.HasLoggedErrors;

```

Comment:
That's not what ContinueOnError does. It only controls whether failed tasks cause the build script to crash. I don't know for sure about this, but I think that uncaught exceptions cause the script to crash regardless of the ContinueOnError setting.